test/factories/api_docs/service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+Factory.define(:api_docs_service, class: ApiDocs::Service) do |factory|
+  factory.sequence(:name) { |n| "service#{n}" }
```

Comment:
Should it be named `service`?

Reply:
It can be anything you want, baby :)

Reply:
I passed this conversation to https://github.com/3scale/porta/pull/129#discussion_r225095820 😄

Reply:
No worries about this, if after that PR it still has that name, i will change it in the next PR about this mentioned in https://github.com/3scale/porta/pull/132#discussion_r225111927
But not urgent 😉